.travis.yml

```diff
@@ -26,6 +32,7 @@ before_install:
   - travis_retry composer self-update && composer --version
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" --prefer-source --no-update -n; fi;
```

Comment:
Do I understand correctly that the next line is executed nevertheless? Doesn't the next line undo the changes here?